task-rest_bold.psyexp

```diff
@@ -78,7 +78,7 @@
         <Param val="time (s)" valType="str" updates="None" name="startType"/>
         <Param val="0.0" valType="code" updates="None" name="startVal"/>
         <Param val="duration (s)" valType="str" updates="None" name="stopType"/>
-        <Param val="1200" valType="code" updates="constant" name="stopVal"/>
+        <Param val="1206" valType="code" updates="constant" name="stopVal"/>
```

Comment:
Am I correctly understanding you are adding 3 seconds for the fixation point at the beginning and end?

Reply:
Yes, exactly.